inference-engine/tests/ie_test_utils/common_test_utils/file_utils.hpp

```diff
@@ -115,4 +116,67 @@ inline bool directoryExists(const std::string &path) {
     return false;
 }
 
+inline void directoryFileListRecursive(const std::string& name, std::vector<std::string>& file_list) {
+    struct CloseDir {
+        void operator()(DIR* d) const noexcept {
+            if (d) {
+                closedir(d);
+            }
+        }
+    };
+    using Dir = std::unique_ptr<DIR, CloseDir>;
+    Dir directory(opendir(name.c_str()));
+    struct dirent *entire;
+    if (directory) {
+        const std::string current_dir{"."};
+        const std::string parent_dir{".."};
+        while ((entire = readdir(directory.get())) != nullptr) {
+            if (std::string(entire->d_name) == parent_dir || std::string(entire->d_name) == current_dir) {
```

Comment:
https://en.cppreference.com/w/cpp/string/basic_string/operator_cmp -> ver 8

Reply:
Unfortunately also since C++20, but we are limited by C++11 (or 14?) because some OSs and compilers compatibility

Reply:
I see
> Until C++20 
```
template< class CharT, class Traits, class Alloc >
bool operator==( const std::basic_string<CharT,Traits,Alloc>& lhs,
                 const CharT* rhs );
```

Reply:
Yeah, it really works in older standards also. Fixed